src/app/hsapp.json

```diff
@@ -2,14 +2,5 @@
   "name": "Example App",
   "description": "An example private app that contains a single card extension.",
   "scopes": ["crm.objects.contacts.read", "crm.objects.contacts.write"],
-  "public": false,
-  "extensions": {
```

Comment:
This is replaced by new naming conventions:
- app.functions -> app.extensions
- example-card.js -> example-card.card.js